kolibri/core/assets/src/views/language-switcher/LanguageSwitcherList.vue

```diff
@@ -73,14 +86,28 @@
         }
         return this.windowBreakpoint;
       },
+      numSelectableLanguages() {
+        return selectableLanguages.length;
+      },
       buttonLanguages() {
-        const prioritized_languages = ['en', 'ar', 'es-419', 'hi-in', 'fr-fr', 'sw-tz'];
-        return prioritized_languages
-          .filter(lang => availableLanguages[lang] !== undefined)
-          .filter(lang => lang !== currentLanguage)
-          .map(lang => availableLanguages[lang])
-          .slice(0, this.numVisibleLanguages)
-          .sort(this.compareLanguages);
+        let buttonLanguages = selectableLanguages;
+        if (buttonLanguages.length > this.numVisibleLanguages + 1) {
+          buttonLanguages = buttonLanguages
+            .sort((a, b) => {
```

Comment:
Seeing some unexpected behaviors here that I'm looking into. Might be because `.sort` is in-place, and references `selectableLanguages`. Will investigate